install/management/appliance.go

```diff
@@ -171,7 +171,8 @@ func (d *Dispatcher) createApplianceSpec(conf *configuration.Configuration) (*ty
 			&types.OptionValue{Key: "guestinfo.vch/sbin/imagec", Value: "-debug -logfile=/var/log/vic/imagec.log -insecure"},
 			&types.OptionValue{Key: "guestinfo.vch/sbin/port-layer-server",
 				Value: fmt.Sprintf("--host=localhost --port=8080 --insecure --sdk=%s --datacenter=%s --cluster=%s --pool=%s --datastore=%s --network=%s --vch=%s",
-					conf.TargetPath, conf.DatacenterName, conf.ClusterPath, conf.ResourcePoolPath, conf.ImageStorePath, conf.ExtenalNetworkPath, conf.DisplayName)},
+					conf.TargetPath, conf.DatacenterName, conf.ClusterPath, conf.ResourcePoolPath+`/`+conf.DisplayName,
```

Comment:
Nice job! 
For this change, I would suggest to leverage the fmt.Sprintf previously.
from "--pool=%s" to "--pool=%s/%s", and just put conf.ResourcePoolPath and conf.DisplayName separately.